gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -52,11 +52,12 @@ public void apply(Project project) {
                 java.removeUnusedImports();
                 // use empty string to specify one group for all non-static imports
                 java.importOrder("");
-                java.trimTrailingWhitespace();
 
                 if (eclipseFormattingEnabled(project)) {
                     java.eclipse().configFile(project.file(eclipseXml.toString()));
                 }
+
+                java.trimTrailingWhitespace();
```

Comment:
this is gonna be different in an IDE vs from gradlew??

Reply:
Technically, but formatting from within Intellij didn't produce the whitespace. I suspect that's due to how IntelliJ automatically trims trailing whitespace upon saving.